nsedt/reports.py

```diff
@@ -35,8 +39,11 @@ def get_market_activity_report(date: str):
 
     cookies = get_cookies()
     url = f"{REPORT_URL}{MARKET_ACTIVITY_REPORT}{date}.csv"
-    return fetch_csv(url, cookies, response_type="raw")
-
+    response = fetch_csv(url, cookies, response_type="raw")
+    df = pd.read_csv(BytesIO(response), skiprows=8)
+    df.drop(columns=['Unnamed: 0'], inplace=True)
+    df = df[:77]
```

Comment:
@shivam2502 Hey, this screws the top gainer/loser when you slice, please can you get rid of it. Thanks.